main.go

```diff
@@ -582,7 +590,15 @@ The validate command expects a configuration file as its only argument. Local fi
 			if err != nil {
 				return fmt.Errorf("invalid path %s", destination)
 			}
-			cfg, err := elementalConfig.ReadConfigRun("/etc/elemental")
+			config, err := config.Scan(collector.Directories(configScanDir...), collector.NoLogs)
+			if err != nil {
+				return err
+			}
+			cc, err := config.String()
+			if err != nil {
+				return err
+			}
+			cfg, err := elementalConfig.ReadConfigRunFromCloudConfig(cc)
```

Comment:
but the pattern seems to be similar so yeah I think this method, or a similar one should be able to consume config directly

Reply:
yeah, this needs another refactor afterwards to merge them more and more and consolidate stuff.